tests/system/src/test/java/test/robot/javafx/scene/NodeInitializationBackgroundThreadTest.java

```diff
@@ -0,0 +1,906 @@
+            c.setValue(Color.RED);
+            c.prefHeight(-1);
+            c.setValue(Color.BLACK);
+            c.prefWidth(-1);
+            accessControl(c);
+        });
+    }
+
+    @Test
+    public void comboBox() {
+        test(() -> {
+            ComboBox c = new ComboBox();
+            c.setSkin(new ComboBoxListViewSkin(c));
+            c.getItems().setAll("ComboBox", "1", "2");
+            return c;
+        }, (c) -> {
+            c.setEditable(true);
+            c.getItems().setAll("ComboBox", nextString(), "2");
+            c.getSelectionModel().select(0);
+            accessControl(c);
+            c.show(); // does not fail here
+        });
+    }
+
+    @Disabled("JDK-8349004") // FIX
+    @Test
+    public void datePicker() {
+        test(() -> {
+            DatePicker c = new DatePicker();
+            c.setSkin(new DatePickerSkin(c));
+            return c;
+        }, (c) -> {
+            c.show(); // fails here
+            c.setValue(LocalDate.now());
+            c.prefHeight(-1);
+            c.setValue(LocalDate.EPOCH);
+            c.prefWidth(-1);
+            accessControl(c);
+        });
+    }
+
+    @Disabled("JDK-8347392") // FIX
+    @Test
+    public void hyperlink() {
+        test(() -> {
+            Hyperlink c = new Hyperlink("Hyperlink");
+            c.setSkin(new HyperlinkSkin(c));
+            return c;
+        }, (c) -> {
+            c.setVisited(true);
+            accessControl(c);
+        });
+    }
+
+    @Disabled("JDK-8347392") // FIX
+    @Test
+    public void label() {
+        test(() -> {
+            Label c = new Label("Label");
+            c.setSkin(new LabelSkin(c));
+            return c;
+        }, (c) -> {
+            c.setLabelFor(c);
+            accessControl(c);
+        });
+    }
+
+    @Disabled("JDK-8349091") // FIX
+    @Test
+    public void lineChart() {
+        test(() -> {
+            LineChart c = new LineChart(createNumberAxis("x"), createNumberAxis("y"));
+            c.getData().setAll(createNumberSeries());
+            return c;
+        }, (c) -> {
+            c.getData().setAll(createNumberSeries());
+            accessChart(c);
+        });
+    }
+
+    @Test
+    public void listView() {
+        test(() -> {
+            ListView c = new ListView();
+            c.setSkin(new ListViewSkin(c));
+            return c;
+        }, (c) -> {
+            c.getItems().setAll("ListView", "1", "2");
+            c.getSelectionModel().select(0);
+            accessControl(c);
+        });
+    }
+
+    @Disabled("JDK-8349096") // FIX
+    @Test
+    public void menuButton() {
+        test(() -> {
+            MenuButton c = new MenuButton();
+            c.setSkin(new MenuButtonSkin(c));
+            return c;
+        }, (c) -> {
+            c.getItems().setAll(new MenuItem("MenuButton"));
+            c.setPopupSide(Side.RIGHT);
+            accessControl(c);
+            c.show();
```

Comment:
This is another case of a `show` method that we should consider changing to require access only on the FX app thread.